app/views/admin/hackathons/show.html.erb

```diff
@@ -27,7 +27,7 @@
         <% if @hackathon.banner.present? %>
           <div>
             <h2 class="heading--small">Banner</h2>
-            <%= image_tag @hackathon.banner, height: 128, style: "background-color: black;" %>
+            <%= image_tag @hackathon.banner.variant(:small), height: 128 %>
```

Comment:
`style: "background-color: black;"` is useful in case the logo is white with a transparent background. There likely is a better way to solve this problem

Reply:
Maybe just have a light gray background around both images?

Reply:
that works! another solution is to create a "preview" of what the card would look like

![image](https://github.com/hackclub/hackathons-backend/assets/20099646/ebdeec31-c556-4a4b-971a-0bca63ef8596)

Reply:
I would do that in a separate PR.

Reply:
Let's keep some sort of background for now. afterwards, it's good to merge